pycloudlib/cloud.py

```diff
@@ -34,6 +35,13 @@ class ImageType(enum.Enum):
     PRO_FIPS = "Pro FIPS"
 
 
+@dataclasses.dataclass
+class ImageInfo:
```

Comment:
Even fancier than NamedTuple. +1 py 3.7 support where dataclass was introduced is good enough for me. Makes the declaration of this  datatype easier to read/consume.

Reply:
Note if pycloudlib requires python 3.7 ++ we need to update our project docs which state python 3.6 https://pypi.org/project/pycloudlib/

Reply:
Since this could represent a breaking change due to py 3.7 features, we should bump the major version unless we can find py 3.7 features that are already in pycloudlib.